packages/account-v2/src/pages/ConnectedApps/ConnectedAppsSidebar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { SideNote } from '@deriv-com/ui';
+
+export const ConnectedAppsSidebar = () => {
+    const openAPIManagingWebsite = () => {
+        window.open(
+            'https://community.deriv.com/t/api-tokens-managing-access-on-third-party-applications-and-mobile-apps/29159',
+            '_blank',
+            'noopener'
```

Comment:
Lets use `'noopener,noreferrer'` . This enhances security and privacy

Reply:
Hi @utkarsha-deriv and @likhith-deriv ,
I don't know things works here 😁  but isn't it good to put this URL in some constants file? 

Maybe you can ignore this if its not relevant. 😄
Thank you 🙇